Readme.md

```diff
@@ -67,23 +67,37 @@ you should use for your Swift version.
 To integrate using Apple's Swift package manager, add the following as a dependency to your `Package.swift`:
 
 ```swift
-.Package(url: "https://github.com/Moya/Moya.git", majorVersion: 8)
+.package(url: "https://github.com/Moya/Moya.git", .branch("10.0.0-dev"))
 ```
 
-and then specify `.Target(name: "Moya")` as a dependency of the Target in which you wish to use Moya.
+and then specify `"Moya"` as a dependency of the Target in which you wish to use Moya.
+If you want to use reactive extensions, add also `"ReactiveMoya"` or `"RxMoya"` as your Target dependency respectively.
 Here's an example `PackageDescription`:
 
 ```swift
+// swift-tools-version:4.0
 import PackageDescription
 
 let package = Package(
-    name: "MyApp",
+    name: "MyPackage",
+    products: [
+        .library(
+            name: "MyPackage",
+            targets: ["MyPackage"]),
+    ],
     dependencies: [
-        .Package(url: "https://github.com/Moya/Moya.git", majorVersion: 8)
+        .package(url: "https://github.com/Moya/Moya.git", .branch("10.0.0-dev"))
+    ],
+    targets: [
+        .target(
+            name: "MyPackage",
+            dependencies: ["Moya", "ReactiveMoya"])
     ]
 )
 ```
 
+Note that as of Moya 10, SPM only works with Swift 4 and greater.
```

Comment:
I know that you need to have Swift 4 on your machine, but can you use Swift 3.2 anyways? 🤔